block-production/tests/mod.rs

```diff
@@ -27,6 +31,8 @@ pub const ACCOUNT_SECRET_KEY: &str =
 
 const STAKER_ADDRESS: &str = "NQ20TSB0DFSMUH9C15GQGAGJTTE4D3MA859E";
 
+const VOLATILE_ENV: bool = false;
```

Comment:
I would consider leaving the default in `true` since most of our tests run in a volatile environment